### PR TITLE
 230.Add decorator functions to pass click options correctly

### PR DESCRIPTION
**Pull request by singhd789**
- this was needed in order to import fre make scripts successfully when creating pytests

## Checklist before requesting a review

- [x] I ran my code
- [x] I tried to make my code readable
- [x] I tried to comment my code
- [ ] I wrote a new test, if applicable
- [ ] I wrote new instructions/documentation, if applicable
- [ ] I ran pytest and inspected it's output
- [x] I ran pylint and attempted to implement some of it's feedback

### CODE REVIEW

File: fre/make/createCheckout.py [hunk 1]
```diff
@@ -5,11 +5,10 @@
 import logging
 import sys
 import click
-from .gfdlfremake import varsfre, yamlfre, checkout, targetfre
 import fre.yamltools.combine_yamls as cy
+from .gfdlfremake import varsfre, yamlfre, checkout, targetfre
 
-@click.command()
-def checkout_create(yamlfile,platform,target,no_parallel_checkout,jobs,execute,verbose):
+def _checkout_create(yamlfile,platform,target,no_parallel_checkout,jobs,execute,verbose):
```
Comment (ilaflott) [inline fre/make/createCheckout.py:11]: backwards! this should not have an underscore. Line 107 should

File: fre/make/createCompile.py [hunk 2]
```diff
@@ -9,8 +9,7 @@
 from .gfdlfremake import varsfre, yamlfre, targetfre, buildBaremetal
 import fre.yamltools.combine_yamls as cy
 
-@click.command()
-def compile_create(yamlfile,platform,target,jobs,parallel,execute,verbose):
+def _compile_create(yamlfile,platform,target,jobs,parallel,execute,verbose):
```
Comment (ilaflott) [inline fre/make/createCompile.py:12]: see other comment

File: fre/make/createDocker.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
 import fre.yamltools.combine_yamls as cy
 
 @click.command()
-def dockerfile_create(yamlfile,platform,target,execute):
+def _dockerfile_create(yamlfile,platform,target,execute):
```
Comment (ilaflott) [inline fre/make/createDocker.py:12]: see above

**Review by ilaflott — CHANGES_REQUESTED** [review]
underscore convention is backwards!

File: fre/make/createDocker.py (hunk 3 above)
Comment (singhd789) [reply fre/make/createDocker.py:12]: ooooop, my bad. Let me fix that

**Comment by singhd789** [comment]
> underscore convention is backwards!

@ilaflott Hmmm so when I reverse the underscore convention, I keep getting this: 
```
TypeError: Callback is not a command.
```

**Comment by ilaflott** [comment]
> > underscore convention is backwards!
> 
> @ilaflott Hmmm so when I reverse the underscore convention, I keep getting this:
> 
> ```
> TypeError: Callback is not a command.
> ```

since it changes a module name, you need to change the name in other places where that old name is imported, is my guess.

this pattern is absolutely doable, most every other subtool currently has it

**Comment by ilaflott** [comment]
> > > underscore convention is backwards!
> > 
> > 
> > @ilaflott Hmmm so when I reverse the underscore convention, I keep getting this:
> > ```
> > TypeError: Callback is not a command.
> > ```
> 
> since it changes a module name, you need to change the name in other places where that old name is imported, is my guess.
> 
> this pattern is absolutely doable, most every other subtool currently has it

let me note that `fre pp checkout` also has the pattern backwards

**Review by rem1776 — APPROVED** [review]

**Comment by singhd789** [comment]
> let me note that `fre pp checkout` also has the pattern backwards... i.e. i'll get on others case about it!

I think I may have done it with ` fre pp configure-yaml ...` as well. 😅

**Review by ilaflott — APPROVED** [review]

**Pull request merged**